.github/workflows/e2e.yaml

```diff
@@ -55,4 +55,4 @@ jobs:
         curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.22.0/kind-linux-amd64
         chmod +x ./kind
         ./kind delete cluster || true
-        make clean
+        make clean
```

Comment:
You should clean up the state of this PR so that this is not included. It just creates unnecessary noise as this makes no real difference to the core work in this PR. Same goes for the update to .github/workflows/preview.yaml